posthog/async_migrations/test/test_0006_persons_and_groups_on_events_backfill.py

```diff
@@ -205,12 +216,14 @@ def test_deleted_data_persons(self):
         create_person_distinct_id(self.team.pk, "1", str(person.uuid))
         delete_person(person)
 
-        self.assertTrue(run_migration())
+        # the mutation will run as noted by person_properties becoming '{}' instead of ''
+        # but the migration will be marked as false as it will fail the postcheck indicating some investigation is needed into the instance's data
+        self.assertFalse(run_migration())
```

Comment:
doesn't this mean that if we have lots of deleted persons for a team the postcheck will fail?